presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -51,6 +54,64 @@ protected void registerFunctions()
         functionAssertions.getMetadata().addFunctions(extractFunctions(plugin.getFunctions()));
     }
 
+    @Test
+    public void testSpatialPartitions()
+            throws Exception
+    {
+        String kdbTreeJson = makeKDBTreeJson();
+
+        assertSpatialPartitions(kdbTreeJson, "POINT EMPTY", null);
```

Comment:
`assertSpatialPartitioning`?

Reply:
@arhimondr This function tests `spatial_partitions` function, not `spatial_partitioning`, hence, I think the name is appropriate.